README.md

```diff
@@ -42,6 +43,6 @@ Ideally, you need two hosts to run this project:
 
 Now you can proceed to any of the following tasks:
 
- * [Cloning a Satellite server](docs/satellite-clone.md)
+ * [Cloning a Satellite server (or) Migrating a Satellite server from rhel6 to rhel7](docs/satellite-clone.md)
```

Comment:
This makes a migration and cloning seem like two different things... I think we can just keep this as `Cloning a Satellite Server` and then detail the migrating part in the satellite-clone.md page